llvm/cmake/modules/AddLLVM.cmake

```diff
@@ -1524,6 +1528,13 @@ macro(add_llvm_tool_subdirectory name)
   add_llvm_external_project(${name})
 endmacro(add_llvm_tool_subdirectory)
 
+macro(add_custom_linker_flags name)
+  if (LLVM_${name}_LINKER_FLAGS)
+    message(STATUS "Applying ${LLVM_${name}_LINKER_FLAGS} to ${name}")
```

Comment:
I'd omit this to avoid the extra build spam (or make it optional).

Reply:
What did you mean by optional here? Like a CMake flag that enables the message? In general, I find this message to be quite helpful when debug whether targets were applied with custom flags without needing to go into the CMakeCache.txt.

Reply:
I'm going to land this as is, and if this becomes very build spammy, we can figure out other approaches to disable it.

Reply:
An alternative to an option would be to use the `DEBUG` mode instead of `STATUS` so developers can use [`CMAKE_MESSAGE_LOG_LEVEL`](https://cmake.org/cmake/help/latest/variable/CMAKE_MESSAGE_LOG_LEVEL.html#variable:CMAKE_MESSAGE_LOG_LEVEL) to suppress the output if desired.

Reply:
https://github.com/llvm/llvm-project/pull/69497